includes/classes/productPulldown.php

```diff
@@ -56,21 +56,25 @@ public function setCategory(int $category_id)
             return $this;
         }
 
-        public function showModel()
+        public function showModel(bool $status)
         {
-            $this->show_model = true;
+            $this->show_model = $status;
             return $this;
         }
 
-        public function showPrice()
+        public function showPrice(bool $status)
```

Comment:
Perhaps this thought is over-engineering, since the the implementation here is more likely to actually be passing these `$status` params via variables ...
The thought is: is it worth defaulting each of these to `true` to allow less verbosity if variables aren't used?

eg: `$foo->showModel()->showPrice()`
instead of requiring
`$foo->showModel(true)->showPrice(true)`

Reply:
i did look at this prior.  i think the problem is what happens when someone wants to hide the price?  or the model?  do you want to have more methods there? such as `hideModel()` and `hidePrice()`?  by including the boolean, users can more easily specify what they want to do.

Reply:
I see where you're going there.
I was actually thinking `showPrice()` and `showPrice(true)` would both mean true,
and `showPrice(false)` for false.

But, let's just leave it for now. At least with no default it'll force the param to be specified, so the intent will be required.

Reply:
actually the default is in the constructor.  so there is a default. `$this->show_price` is the default; the method `showPrice()` changes the var.